backend/src/main/scala/akka/viz/events/LightSnapshot.scala

```diff
@@ -11,8 +13,10 @@ case class LightSnapshot(
     receivedFrom: Set[(String, String)] = Set()
 ) {
 
-  implicit def ref2String(r: ActorRef): String = r.path.toSerializationFormat
-  implicit def refPair2StringPair(pair: (ActorRef, ActorRef)): (String, String) = (ref2String(pair._1), ref2String(pair._2))
+  implicit def refPair2StringPair(pair: (ActorRef, ActorRef)): (String, String) = {
+    val (actor1, actor2) = pair
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/lustefaniak/akka-viz/file/2004467639/issues/source?bid=3121282&fileBranchId=3122179#l17)